src/transformers/modeling_tf_pytorch_utils.py

```diff
@@ -88,7 +88,10 @@ def convert_tf_weight_name_to_pt_weight_name(tf_name, start_prefix_to_remove="",
 
     # The SeparableConv1D TF layer contains two weights that are translated to PyTorch Conv1D here
     if tf_name[-1] == "pointwise_kernel" or tf_name[-1] == "depthwise_kernel":
-        tf_name[-1] = tf_name[-1].replace("_kernel", ".weight")
+        if tf_name[0] == "cvt":
+            tf_name[-1] = "weight"
+        else:
+            tf_name[-1] = tf_name[-1].replace("_kernel", ".weight")
```

Comment:
If we don't make this change, what is the error that pops up? 

(If possible, I'd like to handle the issue in the modeling file rather than to these shared functions :) )

Reply:
Hi, I corrected the problem directly in the modeling file 😊. This condition was to avoid having weight initialization problems. In the tensorflow implementation I was using `tf.keras.layers.DepthwiseConv2D` instead of `tf.keras.layers.Conv2D` with the `groups` parameter. Therefore the tensorflow weight name was different than the pytorch weight name.